rest/taskana-rest-spring/src/main/java/pro/taskana/user/rest/UserController.java

```diff
@@ -54,24 +61,36 @@ public class UserController {
   public ResponseEntity<UserRepresentationModel> getUser(@PathVariable String userId)
       throws UserNotFoundException, InvalidArgumentException {
     User user = userService.getUser(userId);
-
     return ResponseEntity.ok(userAssembler.toModel(user));
   }
 
   /**
    * This endpoint retrieves multiple Users. If a userId can't be found in the database it will be
    * ignored. If none of the given userIds is valid, the returned list will be empty.
+   * If currentUser is set, the current User from the context will be retrieved as well
    *
    * @title Get multiple Users
    * @param userIds the ids of the requested Users
+   * @param currentUser boolean to indicate whether to fetch the current user or not
```

Comment:
The param "currentUser" is a boolean only logically but not from the implementation standpoint. What do you think about changing the description here? It should be clear which values for currentUser are valid

Reply:
fixed